posts/MarkoB/2023-11-30-hibernate-search-7-1-0-Alpha1.adoc

```diff
@@ -3,6 +3,8 @@ Marko Bekhta
 :awestruct-tags: [ "Hibernate Search", "Lucene", "Elasticsearch", "Releases" ]
 :awestruct-layout: blog-post
 :hsearch-doc-url-prefix: https://docs.jboss.org/hibernate/search/7.1/reference/en-US/html_single/
+:hsearch-getting-started-orm-url-prefix: https://docs.jboss.org/hibernate/search/7.1/getting-started/orm/en-US/html_single/
+:hsearch-getting-started-stanadlone-url-prefix: https://docs.jboss.org/hibernate/search/7.1/getting-started/standalone/en-US/html_single/
```

Comment:
Typo: stanadlone => stanadlone.

Same throughout the whole PR :)

Reply:
> Typo: stanadlone => stanadlone.

I've tried ... but nothing got replaced 😅 it's a highly-typo-prone word 😄 (replaced with `standalone` thanks!)